code/src/main/java/com/onlinecontacttracing/messaging/MessageServlet.java

```diff
@@ -0,0 +1,95 @@
+package com.onlinecontacttracing.messaging;
+
+import com.onlinecontacttracing.messaging.LocalityResource;
+import com.onlinecontacttracing.storage.CustomizableMessage;
+import com.onlinecontacttracing.storage.PositiveUser;
+import com.onlinecontacttracing.storage.PotentialContact;
+import com.onlinecontacttracing.messaging.filters.CheckMessagesForFlags;
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+import com.google.gson.Gson;
+import com.google.appengine.api.datastore.DatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.Query.SortDirection;
+import com.google.appengine.api.datastore.Key;
+import com.google.appengine.api.datastore.Projection;
+import java.util.Collections;
+import java.util.List;
+import java.util.ArrayList;
+import java.io.IOException;
+
+@WebServlet("/messaging")
+public class MessageServlet extends HttpServlet {
```

Comment:
So same as above, it seem strange to me to see this as a servlet, I think it's just a java class.

Reply:
We never use the doGet, so that's a good signal this should just be a class called Message.